src/components/switch/switch.tsx

```diff
@@ -55,12 +58,15 @@ export const SwitchLabel = ( {
 	switchId: string;
 	disabled?: boolean;
 	children: ReactNode;
-	size: 'sm' | 'md' | 'lg';
+	size: 'sm' | 'md';
```

Comment:
Are we deprecating lg size?

Reply:
Size `lg` is not on the Figma mockup final design page. Maybe previously we followed in progress Figma. I've also added the backward compatibility for the size `lg`.